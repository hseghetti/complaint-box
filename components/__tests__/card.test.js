import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../card';

describe('Card Component', () => {
  it('Should render Card', () => {
    const component = renderer.create(
      <Card />
    );
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});