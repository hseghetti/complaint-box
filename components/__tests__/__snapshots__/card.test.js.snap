// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component Should render Card 1`] = `
<textarea
  className="chakra-textarea css-5d6l6w"
  onBlur={[Function]}
  onFocus={[Function]}
/>
`;
